src/Raygun.NetCore.Blazor.Server/Class1.cs

```diff
@@ -1,5 +1,8 @@
 ﻿namespace Raygun.NetCore.Blazor.Server
```

Comment:
I imagine we'll just want to delete this class

Reply:
I thought the same, the `Raygun.Blazor.Server` question is addressed here: https://github.com/MindscapeHQ/raygun4blazor/issues/22